skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
```

Comment:
The current behavior in `0.13` treats as multichannel when the input is 3D **and** `output_shape[2] == image.shape[2]`.  Otherwise it is not treated as multichannel.  I think `multichannel_default` should be updated to match that behavior backwards compatibility.

I will update the deprecation message appropriately assuming the future default will be `False`.  Assuming this goes into release 0.14, deprecation would be 0.16?